packages/widgets/src/Pagination/Pagination.styles.js

```diff
@@ -0,0 +1,45 @@
+import css from 'styled-jsx/css'
+import { colors, spacers } from '@dhis2/ui-constants'
+
+export default css`
+    .container {
+        display: flex;
+        justify-content: flex-end;
+        flex-wrap: wrap;
+        padding: ${spacers.dp12};
```

Comment:
This padding seems like it is on the outside container:

![image](https://user-images.githubusercontent.com/7355199/92463416-14095600-f1cc-11ea-982f-4f85474e83e8.png)

Did we not agree to only apply spacing inside a component? Maybe something else has been decided in the meantime, but I still think that it's a good idea to not having spacing outside of a library component.

Reply:
@cooper-joe I agree with @ismay on this: I think it'd be better to remove this. Would you be OK with too?
Since t's coming from [the design system](https://www.sketch.com/s/2f0dba87-fb82-415b-863d-0c2ea63ee77b/a/pmVrZo), so I'm keeping it here until you confirm.

Reply:
Yeah, I agree with removing this. I think the spec was made before we made that decision. Sorry for the out-of-date spec.

Reply:
Removed in d362a52